src/cli/generator-html.ts

```diff
@@ -190,6 +190,18 @@ const topbarFunc = (TopbarEl: AstNode, ctx:GeneratorContext) => {
     return topbarNode
 }
 
+const footerFunc = (FooterEl: AstNode, ctx:GeneratorContext) => {
+    const el = FooterEl as Footer;
+    const footerNode = new CompositeGeneratorNode()
+
+    footerNode.append(`<div class='footer' style='${generateInlineCSS(el,ctx)}'>`, NL);
```

Comment:
there is a html tag for footers.

Also depending on what we put inside of the footer (contact? links to sections in the page?) that having a paragraph makes much sense (but this can be changed later when we rework the footer)

```suggestion
    footerNode.append(`<footer class='footer' style='${generateInlineCSS(el,ctx)}'>`, NL);
```

Suggestion: Also, what about adding classes to the footer?